docs/theme/announce.html

```diff
@@ -1,5 +1,4 @@
 <!-- the following line is displayed in the announcement bar -->
 <!-- keep length under 164 characters (less HTML tags) to fit on 1280px desktop window -->
-<b>We're live!</b> <a href="https://pydantic.dev/logfire">Pydantic Logfire</a> is out in open beta! 🎉<br>
-Logfire is a new observability tool for Python, from the creators of Pydantic, with great Pydantic support.
-Please try it, and tell us <a href="https://docs.pydantic.dev/logfire/help/">what you think</a>!
+Pydantic Logfire &mdash; has left beta and is ready for use by all developers and organizations.
+Read <a href="https://pydantic.dev/articles/logfire-announcement">the announcement</a>.
```

Comment:
I think we should say something that makes it clear what Logfire is here. E.g.:

```suggestion
Pydantic Logfire &mdash; has left beta and is ready for use by all developers and organizations who need to monitor their applications.
Read <a href="https://pydantic.dev/articles/logfire-announcement">the announcement</a>.
```